app/scripts/components/common/datepicker/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { ReactNode, useState } from "react";
+import { Icon } from "@trussworks/react-uswds";
+import Calendar from 'react-calendar';
+import Tippy from "@tippyjs/react";
+
+import 'react-calendar/dist/Calendar.css';
+import './index.scss';
+
+interface SimpleDatePickerProps {
+  disabled: boolean;
+  tipContent: ReactNode;
+  onConfirm: (date: Date | null) => void;
+  triggerHeadReference: string;
+  selectedDay: Date | null;
+  renderTriggerElement: (props: {
+    onClick: () => void;
+    disabled: boolean;
+    tipContent: ReactNode;
+    triggerHeadReference: string;
+    selectedDay: Date | null;
+  }) => ReactNode;
+}
+
+export const SimpleDatePicker = ({
+  disabled,
+  tipContent,
+  onConfirm,
+  triggerHeadReference,
+  selectedDay,
+  renderTriggerElement
+}: SimpleDatePickerProps) => {
+  const [isCalendarOpen, setIsCalendarOpen] = useState(false);
+
+  const handleDateChange = (date: Date) => {
+    onConfirm(date);
+    setIsCalendarOpen(false);
+  };
+
+  const handleTriggerClick = () => {
+    setIsCalendarOpen(!isCalendarOpen);
```

Comment:
Toggling doesn't seem to work - the function gets called twice in a row.
I think it is because `onClickoutside` function also gets called when the triggerElement is clicked. can you take a look at this ?

Reply:
IMO I think we should keep the default uswds behavior (ie. clicking on the trigger should toggle the calendar)